api/v1/Plan.php

```diff
@@ -1279,7 +1279,7 @@ public function Export(array &$result, ?array &$errors = null): bool
 
                 $l['deleted'] = $this->getDatabase()->query(
                     "
-                    SELECT geometry_id
+                    SELECT geometry_id, 0 as geometry_mspid
```

Comment:
just return 0 for the mspid for "deleted"
This seems to be the default for all the sea basins with starting plans